saga/src/types/clients.d.ts

```diff
@@ -0,0 +1,42 @@
+import { Commands } from './commands';
```

Comment:
Curious why this is required, just better organization?
Also, please make this `.ts` instead of `.d.ts`.

Reply:
this is explicitly defining messaging contracts...where i have been before we defined `.d.ts` files for such API contracts. 
is that what you are asking? I may misunderstand..happy to do something different if you prefer.